packages/banner/jest.config.js

```diff
@@ -5,5 +5,5 @@ module.exports = {
   ...baseConfig,
   displayName: name,
   name: name,
-  testMatch: [`${__dirname}/**/*/?(*.)+(spec|test).js`]
+  testMatch: [`${__dirname}/**/*/?(*.)+(spec|test).(js|ts|tsx)`]
```

Comment:
Are there still js spec files being tested?

Reply:
there aren't .js files, no.  We could remove that. 🤷